edith-step/src/ReceiptInput.js

```diff
@@ -3,27 +3,57 @@ import React from 'react';
 export default class ReceiptInput extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {items: [], itemName: '', itemPrice: 0.0};
+    this.state = {items: [], itemName: '', itemPrice: 0.0, itemQuantity: 1, 
+                  };
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
+    this.getDate = this.getDate.bind(this);
+  }
+
+  getDate() {
+      const date = new Date(Date.now());
+      let month = date.getMonth() + 1;
+      month < 10 ? month = "0" + month.toString() : month = month.toString();
+      let day = date.getDate();
+      day < 10 ? day = "0" + day.toString() : day = day.toString();
+      let year = date.getFullYear();
+      return [year, month, day].join("-");   
```

Comment:
it might be helpful to add a comment stating the date format here.
YYYY-MM-DD ?

Reply:
Yea. I added it